rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/TclLauncherUi.xtend

```diff
@@ -127,6 +126,19 @@ class TclLauncherUi implements Launcher {
 		return true
 	}
 
+	def storeTestParameterAsLastTestExecution(List<String> testCasesCommaList, IProject project, TclLauncher launcher,
+		Map<String, Object> options) {
+		logger.debug("Storing test execution as last launch")
+		val lastTestLaunch = new LastTestLaunchInformation()
+		lastTestLaunch => [
```

Comment:
If TestLanuchInformation is annotated as @Data class, data will be passed to constructor